src/jobs/jobs.module.ts

```diff
@@ -49,6 +42,6 @@ import { CaslModule } from "src/casl/casl.module";
     PoliciesModule,
     OrigDatablocksModule,
   ],
-  providers: [JobsService, JobConfigService, EmailJobActionCreator],
+  providers: [CaslAbilityFactory, JobsService, JobConfigService, EmailJobActionCreator],
```

Comment:
JobModule should not provide CaslAbilityFactory. Services should only be provided by one module IMO.